certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                    "process_name": {
+                      "type": "string",
+                      "description": "Process Name"
+                    },
+                    "process_description": {
+                      "type": "string",
+                      "description": "Process Description"
+                    },
+                    "process_id": {
+                      "type": "string",
+                      "description": "Process ID"
+                    },
+                    "request_id": {
+                      "type": "string",
+                      "description": "Request ID"
+                    },
+                    "status": {
+                      "type": "string",
+                      "description": "Stage Status"
+                    },
+                    "actioned_by": {
+                      "type": "string",
+                      "description": "User who actioned"
+                    },
+                    "date_actioned": {
+                      "type": "string",
+                      "description": "Date ticket was actioned"
+                    },
+                    "stage_name": {
+                      "type": "string",
+                      "description": "Stage Name"
+                    },
+                    "stage_description": {
+                      "type": "string",
+                      "description": "Stage Description"
+                    },
+                    "approvers_emails": {
+                      "type": "string",
+                      "description": "Email addresses of approvers"
+                    },
+                    "view_link": {
+                      "type": "string",
+                      "description": "Link to view tickets"
+                    },
+                    "action_data": {
+                      "type": "object"
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Find a Ticket",
+        "operationId": "FindApprovalStageRequest",
+        "description": "Search for a Ticket by ID.",
+        "parameters": [
+          {
+            "name": "approval_stage_request_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetApprovalStageRequests",
+              "value-path": "id",
+              "value-collection": "msg",
+              "value-title": "name"
+            },
+            "description": "Ticket ID",
+            "x-ms-summary": "Ticket ID"
+          }
+        ]
+      }
+    },
+    "/get_approval_stage_requests": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.